zeebe/broker/src/main/java/io/camunda/zeebe/broker/system/partitions/ZeebePartition.java

```diff
@@ -43,6 +43,7 @@ public final class ZeebePartition extends Actor
         DiskSpaceUsageListener,
         SnapshotReplicationListener {
 
+  public static final String ACTOR_NAME_PREFIX = "Partition";
```

Comment:
:wrench: Hm, now we are mixing two things, the actor name and the name of the health monitoring component. I'd prefer if we keep the actor name as is and use a separate prefix + method to build the component name. That also avoids having to call `ZeebePartition#buildActorName` from `BrokerHealthCheckService`, instead we could call `ZeebePartition#componentName` (or a similarly named method).